dawn/src/dawn/AST/ASTStmt.h

```diff
@@ -49,22 +49,22 @@ struct StmtData {
 class Stmt : public std::enable_shared_from_this<Stmt> {
 public:
   /// @brief Discriminator for RTTI (dyn_cast<> et al.)
-  enum StmtKind {
-    SK_BlockStmt,
-    SK_ExprStmt,
-    SK_ReturnStmt,
-    SK_VarDeclStmt,
-    SK_StencilCallDeclStmt,
-    SK_VerticalRegionDeclStmt,
-    SK_BoundaryConditionDeclStmt,
-    SK_IfStmt
+  enum class Kind {
```

Comment:
Are others ok with this shortening of the enum class name? I figured since this is inside the `Stmt` class, that the prefix is unnecessary.